Python/libraries/recognizers-date-time/recognizers_date_time/date_time/base_date.py

```diff
@@ -534,23 +562,13 @@ def extract_relative_duration_date_with_in_prefix(self, text: str, duration_er:
             if (str.isspace(before_str) or before_str is None) and (str.isspace(after_str) or after_str is None):
                 continue
 
-            match = RegExpUtility.match_end(self.config.in_connector_regex, before_str, True)
-            if match and match.success:
-
-                start_token = match.index
-                range_unit_math = self.config.range_unit_regex.match(text[duration.start: duration.start
-                                                                          + duration.length])
-
-                if range_unit_math:
-                    since_year_match = self.config.since_year_suffix_regex.match(after_str)
-
-                    if since_year_match:
-                        result.append(Token(start_token, duration.end + len(since_year_match)))
+            ers, success = self.extract_in_connector(text, after_str, before_str, duration, True)
```

Comment:
`ers` is the name of a variable? what it represents?

Reply:
`extract_results`, it is used though the code to represent a list of extract_result, i would normally not use it but here i am just creating it because as we can't do `out` params on python, i needed to store the two results of the method i was calling somewhere, it's kind of an unimportant var. i could change it though 🤔

Reply:
The name was kind of uncertain, but if it is used normally with that name, it's ok.